### PR TITLE
Finomantra Version 1

### PR DESCRIPTION
**Pull request by Sachanh**
Hi All,

This is the version one of Finomantra UI. 
Responsiveness can be a problem at some places for mobile view.
Need your suggestion to overcome from there.

Also i need your opinon regarding naming convention.

### CODE REVIEW

**Comment by harshitsaxena586** [comment]
hey Himanshu,here's my review on Your component Library,I hope making these changes would make it even better -

Starting from the homepage of your library :
-So for the hero section, you are using flexbox while it's named as grid class makes it confusing if it's the parent div for holding other stuff just name its class as wrap or container 

-another thing is it's not perfectly aligned in center add `justify-content:center ` in the parent div currently named as grid also adding some padding to the same would look really nice 

-for the heading I noticed there was an H2 tag saying "One-stop solution..." that's white in color not visible please change that 

also I noticed you are using span tag in heading which is not required there for now simply use `<br>` and write the rest of the heading

that's How it would look after the changes 
image

coming to the actual components 
please increase the font size of the description its too small to read rn use 1rem instead of 12px

alerts: add some padding its too thin rn
input:I am expecting you will add something because rn I don't see much styling in them
cards can also be improved in styling 

overall things are left towards the end like slider and a lot of places don't have the code snippets to copy from but again it's great you tried and made it so far ,I am hoping to see more progress soon

File: Component/avatar.html [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+                    <div class="avatar-all-without-badge">
+                        <img class="micro" src="/Assets/boy.png" alt="micro-Avatar">
+                        <img class="micro" src="/Assets/girl.png" alt="micro-Avatar">
+                        <img class="mini" src="/Assets/boy.png" alt="micro-Avatar">
+                        <img class="mini" src="/Assets/girl.png" alt="micro-Avatar">
+                        <img class="small" src="/Assets/boy.png" alt="micro-Avatar">
+                        <img class="small" src="/Assets/girl.png" alt="micro-Avatar">
+
+                    </div>
+                    <!-- <div class="avatar-all-with-badge">
```
Comment (harshith-venkatesh) [inline Component/avatar.html:81]: You can remove the comments if you are not using it

**Review by harshith-venkatesh — APPROVED** [review]

File: Component/avatar.html (hunk 1 above)
Comment (Sachanh) [reply Component/avatar.html:81]: removed the unnecessary commented out things.
Thanks for the suggestions

**Comment by Sachanh** [comment]
As Harshit mentioned refactored my code. Edited home page landing page , font size etc.
Will work on remaining things as per requirement.

**Comment by sakshijainn** [comment]
1. Your desktop version has two places to click for components : components dropdown and the .left-nav navigation bar, what I would suggest disable the dropdown in desktop version and enable in mobile version 
Instead to not make your navbar have nothing: add social icons , Home button  if you want. 

2. The background color :orange of the dropdown components tab looks odd , try to have same color theme across your UI
what I suggest us : match it with the button hover color of left-nav . Also the color of  vertical scrollbar seems off, match it with UI

3.Try to have sticky footer, your footer is taking space according to the content present  in .flex-col

4. For Reference:
image
There is a lot of unnecessary whitespace after navbar and before .row-grid container (white color space),.  Resolving it should uplift your left-nav upwards to see the whole scroll bar!

5. For Reference:
image
some of the components do not have headings like Card one

For Future Refinement  which you can skip for right now :
1. Your library has a very weak CSS in terms of badges /alerts wrt their text style/padding/margin. Try to improve them later
2. Your library has a very less reusability scope: You create html file for all components : How one can use a particular component in his/her library where all the CSS of all components is contained in one single component.css 
You did reverse of things:
We should have one components.html file containing html of all components 
We should have one  components.css importing all the URLs of all components 
We should have a folder of all components  containing their CSS  only- eg: alert.css , badge.css

**Pull request merged**